documentation/src/docs/segment-reader.md

```diff
@@ -0,0 +1,70 @@
+<!--
+Copyright Pravega Authors.
+
+Licensed under the Apache License, Version 2.0 (the "License");
+you may not use this file except in compliance with the License.
+You may obtain a copy of the License at
+
+    http://www.apache.org/licenses/LICENSE-2.0
+
+Unless required by applicable law or agreed to in writing, software
+distributed under the License is distributed on an "AS IS" BASIS,
+WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+See the License for the specific language governing permissions and
+limitations under the License.
+-->
+# Introduction to Segment Reader
+Instead of processing all the data of stream, segment reader provides a way to process the data from a specific segment. To
+create a segment reader, user needs to initialize segment reader manager.
+# How to initialize Segment Reader Manager
+Segment reader manager provides a way to manage segment reader. `getSegmentReaders` API returns list of SegmentReader based 
+on the provided StreamCut. For StreamCut `StreamCut.UNBOUNDED`, it fetches head StreamCut and then returns list of SegmentReader.
+```java
+URI controllerURI = URI.create("tcp://localhost:9090");
+
+ClientConfig clientConfig = ClientConfig.builder().controllerURI(controllerURI).build();
+
+SegmentReaderManager<String> segmentReaderManager = SegmentReaderManager.create(clientConfig, serializer);
+List<SegmentReader<String>> segmentReaderList = segmentReaderManager.getSegmentReaders(Stream.of(scope, stream), StreamCut.UNBOUNDED).join();
+```
+Once user are done with SegmentReaderManager, closes this segment reader manager. No further methods may be called after close.
```

Comment:
May be "Once done with SegmentReaderManager, user closes this segment reader manager."

Reply:
Done